src/components/Header/Settings/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Icon, Popover } from 'antd';
+
+
+/**
+ * Settings selector component.
+ *
+ * @component
+ */
+function Settings()
+{
+    return(
+        <Popover
+            content={<p style={{ color: '#a7425c' }}>Coming soon</p>}
+            trigger="hover"
+        >
```

Comment:
the popover's direction is incorrect. Pay attention to the bottom part of the popover in the image attached.

![Screenshot from 2020-04-17 22-08-58](https://user-images.githubusercontent.com/22552561/79624488-3e148b80-80f8-11ea-89c4-26ee2059f0b4.png)